data/json/construction.json

```diff
@@ -3700,10 +3700,10 @@
     "description": "Build Reinforced Double-Pane Glass Window",
     "//": "Step 1: Build reinforced double-pane glass window",
     "category": "REINFORCE",
-    "required_skills": [ [ "fabrication", 9 ] ],
+    "required_skills": [ [ "fabrication", 3 ] ],
     "time": "120 m",
     "qualities": [ [ { "id": "HAMMER", "level": 2 } ], [ { "id": "SAW_M", "level": 1 } ] ],
-    "components": [ [ [ "reinforced_glass_sheet", 2 ] ], [ [ "nail", 16 ] ], [ [ "2x4", 4 ] ] ],
+    "components": [ [ [ "glass_sheet", 1 ] ], [ "reinforced_glass_sheet", 1 ] ], [ [ "nail", 16 ] ], [ [ "2x4", 4 ] ] ],
```

Comment:
I believe this will generate errors as you've removed the braces, too?

Reply:
this one confuses me as I didn't do it in json, ill fix it

Reply:
"components": [ [ [ "glass_sheet", 1 ] ], [ [ "reinforced_glass_sheet", 1 ] ], [ [ "nail", 16 ] ], [ [ "2x4", 4 ] ] ],